src/system/applications/combat/combatant.ts

```diff
@@ -0,0 +1,54 @@
+import { DocumentModificationOptions } from '@league-of-foundry-developers/foundry-vtt-types/src/foundry/common/abstract/document.mjs';
+import { SchemaField } from '@league-of-foundry-developers/foundry-vtt-types/src/foundry/common/data/fields.mjs';
+import { CosmereActor } from '../../documents';
+import { ActorType, TurnSpeed } from '@src/system/types/cosmere';
+
+/**
+ *
+ */
+
+export default class CosmereCombatant extends Combatant {
+    //sets actor type to CosmereActor, to get typescript to stop yelling. Might be a better way to do this but I'm newish to TS and don't know how.
+    override get actor(): CosmereActor {
+        return super.actor as CosmereActor;
+    }
+
+    //on creation, combatants turn speed is set to slow, activation status to false, and then sets the initiative, bypassing the need to roll
+    protected override _onCreate(
+        data: SchemaField.InnerAssignmentType<DataSchema>,
+        options: DocumentModificationOptions,
+        userID: string,
+    ) {
+        super._onCreate(data, options, userID);
+        void this.setFlag('cosmere-rpg', 'turnSpeed', 'slow');
+        void this.setFlag('cosmere-rpg', 'activated', false);
+        void this.combat?.setInitiative(
+            this.id!,
+            this.generateInitiative(
+                this.actor.type as ActorType,
+                TurnSpeed.Slow,
```

Comment:
Looking at this I realise I hadn't re-declared the type for actors to ActorType yet. If you add the following to the top of the CosmereActor class you can get rid of all the `as ActorType` casts. 

```ts
    // Redeclare `actor.type` to specifically be of `ActorType`.
    // This way we avoid casting everytime we want to check/use its type
    declare type: ActorType;
```

See system/documents/item for reference.